src/views/Dog/Dog.js

```diff
@@ -17,5 +18,17 @@ export default function Dog(props) {
     fetchData();
   }, [id]);
 
-  return <div>{loading ? <p>...page is loading</p> : <DogDetail {...dog} />}</div>;
+  const history = useHistory();
+
+  const handleDelete = async (e) => {
```

Comment:
I might also check for an error on delete

Reply:
added!